internal/diskutil.go

```diff
@@ -364,6 +364,33 @@ func GetMatchingSymlinksInDirs(path string, dirs ...string) ([]string, error) {
 	return links, nil
 }
 
+// GetOrphanedSymlinks returns the devices that were symlinked previously, but didn't match the updated
+// LocalVolumeSet filter
+func GetOrphanedSymlinks(symlinkDir string, validDevices []BlockDevice) ([]string, error) {
+	orphanedSymlinkDevices := []string{}
+	paths, err := FilePathGlob(filepath.Join(symlinkDir, "/*"))
+	if err != nil {
+		return orphanedSymlinkDevices, err
+	}
+
+PathLoop:
+	for _, path := range paths {
+		for i, device := range validDevices {
+			isMatch, err := PathEvalsToDiskLabel(path, device.KName)
+			if err != nil {
+				return orphanedSymlinkDevices, err
+			}
+			if isMatch {
+				continue PathLoop
+			} else if i == len(validDevices)-1 {
+				orphanedSymlinkDevices = append(orphanedSymlinkDevices, path)
+			}
```

Comment:
this code is bit hard to read. As I understand - the else block checks if `path` was not found in `validDevices`. Can we rewrite this code like:

```golang
for _, path := range paths {
  found := false
  for _, device := range valiDevices {
    if isMatch {
      found = true
      break
   }
   if !found {
      orphanedSymlinkDevices = append(orphanedSymlinkDevices, path)
  }
}
```

Which IMO is bit easier to read?

Reply:
Thanks for the suggestion.  Updated the method.